packages/pigment-css-react/src/Container.d.ts

```diff
@@ -0,0 +1,29 @@
+import { Breakpoint } from './base';
+import { PolymorphicComponent } from './Box';
+
+type ContainerBaseProps = {
+  /**
+   * If `true`, the left and right padding is removed.
+   * @default false
+   */
+  disableGutters?: boolean;
+  /**
+   * Set the max-width to match the min-width of the current breakpoint.
+   * This is useful if you'd prefer to design for a fixed set of sizes
+   * instead of trying to accommodate a fully fluid viewport.
+   * It's fluid by default.
+   * @default false
+   */
+  fixed?: boolean;
```

Comment:
Although I've added it here, I did not find this actually being used anywhere in the `createContainer` code. So it can be removed.

Reply:
I found it here https://github.com/mui/material-ui/blob/a1f89cea988839f5e305ac705180dc9f4023881a/packages/mui-system/src/Container/createContainer.tsx#L93. I'd keep it as well as https://github.com/mui/material-ui/blob/a1f89cea988839f5e305ac705180dc9f4023881a/packages/mui-system/src/Container/createContainer.tsx#L106-L124

Reply:
Done